modules/files.nix

```diff
@@ -59,13 +59,18 @@ in
             targetPath="$HOME/$relativePath"
             if [[ -e "$targetPath" \
                 && ! "$(readlink "$targetPath")" == ${homeFilePattern} ]] ; then
-              errorEcho "Existing file '$targetPath' is in the way"
-              collision=1
+              if [[ -n "$HOME_MANAGER_BACKUP_EXT" ]] ; then
```

Comment:
Have you tried running without specifying the `-b` option? I'm wondering whether this won't error out. Perhaps you need `-v HOME_MANAGER_BACKUP_EXT && -n $HOME_MANAGER_BACKUP_EXT`?

Reply:
Hmm, just thought of the case where there already is a file whose name matches the backup file name. Right now the existing file would be silently overwritten?

Reply:
Yeah, that's the kind of issue I'm still chewing on. Appreciating the second pair of eyes here. I think probably that should be treated as a collision, and the user can either delete the old backup or use a different extension.

As to `-v` - I don't think that's necessary. An unset variable will default to an empty string, an -n will reject it.

Reply:
Agreed about `-v`. I had the idea that these small scripts also use `set -euo pipefail`. I'll fix that in a separate commit.